multi_import/multi_importer.py

```diff
@@ -104,13 +102,11 @@ def import_datasets(self, datasets):
         new_objects_refs = {}
 
         for importer in self.import_exporters:
-            filename, dataset = datasets.get(importer.model, (None, None))
-            if not filename:
-                continue
-
-            result = importer.generate_import_diff(dataset, new_objects_refs)
-            new_objects_refs[result.model] = result.new_object_refs
-            results.add_result(filename, result)
+            for file_data in datasets.get(importer.model, []):
+                filename, data = file_data
+                result = importer.generate_import_diff(data, new_objects_refs)
+                new_objects_refs[result.model] = result.new_object_refs
```

Comment:
We're going to need to merge multiple new_object_refs if they're from the same model.